srcpkgs/font-Siji/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'font-Siji'
+pkgname=font-Siji
+version=0.0.180725
+revision=1
+build_style=fetch
+hostmakedepends="git"
+short_desc="An iconic bitmap font based on Stlarch with additional glyphs"
+maintainer="xaltsc <xaltsc@protonmail.ch>"
+license="GPL-2.0"
```

Comment:
this is not a correct SPDX short-identifier, it is missing either -only or -or-later, in this case it is -or-later